src/main/java/stepsdesign/ControlCenterSteps.java

```diff
@@ -0,0 +1,205 @@
+  public void createButtonControlCenterPage() {
+    CdfControlCenterActions.clickOnCreateButtonControlCenterPage();
+  }
+
+  @Then("Verify the pipeline created successfully is present in control center page")
+  public void pipelinePresentControlCenterPage() {
+    CdfControlCenterActions.pipelinePresentControlCenterPage();
+  }
+
+  @Then("Click on the delete icon of the created pipeline and pipeline should get deleted successfully")
+  public void deleteControlCenterPipeline() {
+    CdfControlCenterActions.deletePipelineControlCenter();
+  }
+
+  @Then("Verify the deleted pipeline is not present in the control center page")
+  public void pipelineDeletedIsNotPresent() {
+    CdfControlCenterActions.pipelineDeletedIsNotPresent();
+  }
+
+  @Then("Verify the user is able to set the preferences for the created pipeline in the control center page")
+  public void setPreferencesForPipeline() {
+    CdfControlCenterActions.clickOnSetPreferencesIcon(); }
+
+  @Then("Verify the user is able to enter the value in the key input field {string}")
+  public void setPreferencesKeyInputValue(String value) {
+    CdfControlCenterActions.enterKeyInputValue(value); }
+
+  @Then("Verify the user is able to enter the value of the key in the value input field {string}")
+  public void setPreferencesValueInput(String value) {
+    CdfControlCenterActions.enterValueInput(value);
+  }
+
+  @Then("Verify user is able to click on save and close button of set preferences")
+  public void clickSaveAndCloseButton() {
+    CdfControlCenterActions.clickOnSaveAndCloseButton(); }
+
+  @Then("Verify user is able to click on the data pipeline added in the control center page")
+  public void clickOnDataPipelineTab() {
+    CdfControlCenterActions.clickOnDataPipelineTab(); }
+
+  @Then("Verify user is able to click on the delete icon of preferences to delete the added preferences successfully")
+  public void clickOnDeleteIconSetPreferences() {
+    CdfControlCenterActions.clickOnDeleteIconSetPreferences(); }
+
+  @Then("Verify user is able to click on the dataset entity icon to see the details page of the dataset")
+  public void clickOnDatasetEntityIcon() {
+    CdfControlCenterActions.clickOnDatasetEntityIcon(); }
+
+  @Then("Verify user is navigated to the details page of the dataset entity icon successfully")
+  public void navigateDetailsPageDatasetEntity() {
+
+    CdfControlCenterActions.navigateDetailsPageDatasetEntity();
+  }
+
+  @Then("Click on the delete icon of the dataset entity and the dataset should get deleted successfully")
+  public void deleteDatasetEntity() {
+    CdfControlCenterActions.clickOnDeleteDatasetEntity(); }
+
+  @Then("Verify the deleted dataset {string} entity is not present in control center page")
+  public void deletedDatasetIsNotPresent(String value) {
+    CdfControlCenterActions.deletedDatasetNotPresent(value);
+  }
+
+  @Then("Click on the truncate icon of the dataset entity and the dataset should get truncated successfully")
```

Comment:
Click truncate and verify the successful truncate of dataset entity

Reply:
changed